benchmark/Grafana-dashboard.json

```diff
@@ -68,7 +37,7 @@
     {
       "datasource": {
         "type": "prometheus",
-        "uid": "${DS_PROMETHEUS}"
+        "uid": "${datasource}"
```

Comment:
This diff is a little weird. I'm not sure if this happened because you missed a checkbox when you exported it or not.

Reply:
<img width="849" height="398" alt="image" src="https://github.com/user-attachments/assets/c409f044-df92-4ecc-b85b-686fbf02ea4c" />